ioctl/newcmd/account/account.go

```diff
@@ -88,10 +84,8 @@ func NewAccountCmd(client ioctl.Client) *cobra.Command {
 
 	_flagEndpointUsage, _ := client.SelectTranslation(_flagEndpoint)
 	_flagInsecureUsage, _ := client.SelectTranslation(_flagInsecure)
-
-	ac.PersistentFlags().StringVar(&endpoint, "endpoint", client.Config().Endpoint, _flagEndpointUsage)
-	ac.PersistentFlags().BoolVar(&insecure, "insecure", !client.Config().SecureConnect, _flagInsecureUsage)
-
+	client.SetEndpointWithFlag(ac.PersistentFlags().StringVar, _flagEndpointUsage)
```

Comment:
if you want to create this flag inside of client, passing `ac.PersistentFlags()` to client via `CreateFlagVars`.
Also, _flagEndpointUsage and _flagInsecureUsage are not command dependent, which should be defined in client

Reply:
I see, thx. 
I'v done!